opta/commands/destroy.py

```diff
@@ -88,16 +90,13 @@ def destroy(
 
     tf_flags: List[str] = []
     if auto_approve:
-        # Note that for ci, you can just do "yes | opta destroy --auto-approve"
-        click.confirm(
-            fmt_msg(
-                f"""
-                Are you REALLY sure you want to run destroy with auto-approve?
-                ~Please make sure *{layer.name}* is the correct opta config.
-                """
-            ),
-            abort=True,
+        logger.info(
+            f"{fg('red')}{attr('bold')}Opta will now destroy the {attr('underlined')}{layer.name}{attr(0)}"
+            f"{fg('red')}{attr('bold')} layer.{attr(0)}\n"
+            f"{fg('red')}{attr('bold')}Sleeping for {attr('underlined')}2 secs{attr(0)}"
+            f"{fg('red')}{attr('bold')}, if user wants to abort.{attr(0)}"
```

Comment:
why red? it is not an error, just an info message.
can you also tell the user how to exit ?
`Press Ctrl-C to abort now.`